src/bg_daemon/main.m

```diff
@@ -0,0 +1,50 @@
+#import <Foundation/Foundation.h>
+#import <AppKit/AppKit.h>
+
+int main(int argc, const char * argv[]) {
+  int ret = 0;
+
+  @autoreleasepool {
+    NSUserDefaults *args = [NSUserDefaults standardUserDefaults];
+
+    NSError *error=nil;
+    NSString *backgroundPicture = nil;
+    NSWorkspace *workspace = [NSWorkspace sharedWorkspace];
+    BOOL backgroundChanged = NO;
+
+    backgroundPicture = [args stringForKey:@"i"] ? [args stringForKey:@"i"] : [args stringForKey:@"-input"];
+
+    // ghetto way to check if help is on or not
+    if ([[[NSProcessInfo processInfo] arguments] containsObject:@"-h"] ||
+        [[[NSProcessInfo processInfo] arguments] containsObject:@"--help"]){
+      fprintf(stdout, "quack: update your desktop background\n");
+      fprintf(stdout, "-i (--input) - input filepath to list of frames "\
```

Comment:
I like this, what about removing the -i flag?
```bash
$ quack [FILENAME] [-h/help]
```

I think the main function of this command is to change the background image, so having a flag for such images would be a little redundant. 

Thoughts?